tests/foreman/ui/test_contentview.py

```diff
@@ -17,6 +17,71 @@
 
 from robottelo.constants import FAKE_FILE_NEW_NAME, REPOS, DataFile
 
+VERSION = 'Version 1.0'
+
+
+@pytest.mark.tier2
+@pytest.mark.upgrade
+def test_positive_end_to_end(session, module_target_sat, module_org, module_lce, target_sat):
```

Comment:
```suggestion
def test_positive_end_to_end(session, module_target_sat, module_org, module_lce):
```
Common, don't be greedy, use just one of them here and bellow.